apps/sample-imdb/app/movie/[id]/page.tsx

```diff
@@ -0,0 +1,80 @@
+import { HeaderNav } from '~/components/header-nav'
+import { Rating } from '~/components/ratings'
+import { fetchDefaultTitles, getMovie } from '~/lib/db.server'
+import { TitlesRecord } from '~/lib/xata.codegen.server'
+
+export default async function Movie({ params }: { params: { id: string } }) {
+  const { id } = params
+  const {
+    coverUrl = '',
+    primaryTitle,
+    summary,
+    startYear,
+    genres = [],
+    runtimeMinutes,
+    averageRating,
+    numVotes,
+  } = (await getMovie(id)) as TitlesRecord
```

Comment:
Why do we need this `as TitlesRecord`? 🤔

Reply:
I fixed it. when it's `null` I'm throwing a `404` and now I don't need the assertion anymore.